Use_Cases/SSH.md

```diff
@@ -61,7 +61,7 @@ if [ ! -z "$SSH_TTY" ]; then
     #     RemoteForward /home/user/.gnupg/S.gpg-agent.ssh /home/user/.gnupg/S.gpg-agent
     if [ -e $HOME/.gnupg/S.gpg-agent.ssh ]; then
         mv $HOME/.gnupg/S.gpg-agent{.ssh,}
-    elif [ -e "/var/run/user/$(id -u)/gnupg/S.gpg-agent" ]; then
+    elif [ -e "/var/run/user/$(id -u)/gnupg/S.gpg-agent.ssh" ]; then
```

Comment:
@lrvick either this addition is correct or the line below this one needs to be changed to use `{,.ssh}` instead.